pep-0008.txt

```diff
@@ -914,12 +914,14 @@ older convention of prefixing such globals with an underscore (which
 you might want to do to indicate these globals are "module
 non-public").
 
-Function Names
+Function and variable names
 ~~~~~~~~~~~~~~
```

Comment:
Oops, sorry, these squigglies need to be extended.  They should underline the full header text.